docs/team/ivanwong.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Ivan's Project Portfolio Page
+---
+
+### Project: AddressBook Level 3
+
+AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```

Comment:
Should the project name and description be changed to fit TutorAid instead?

Reply:
Yes! Changes made, thanks.